packages/use-dataloader/src/useDataLoader.js

```diff
@@ -51,15 +51,16 @@ const useDataLoader = (
     pollingInterval,
   } = {},
 ) => {
-  const { addCachedData, addReload, clearReload, getCachedData } =
-    useDataLoaderContext()
+  const dataLoaderContext = useDataLoaderContext()
```

Comment:
I think you can destructure both functions you are using from `useDataLoaderContext()`:

```js
  const { addCachedData, getCachedData } = useDataLoaderContext()
```